code/modules/reagents/reagent_containers.dm

```diff
@@ -35,6 +35,10 @@
 /obj/item/reagent_containers/Destroy()
 	return ..()
 
+/obj/item/reagent_containers/attack(mob/living/M, mob/living/user, params)
+	if (user.combat_mode)
+		return ..()
```

Comment:
```suggestion
/obj/item/reagent_containers/attack(mob/living/M, mob/living/user, params)
	if (!user.combat_mode)
		return
	return ..()
```
This does the same, but it's a bit more explicit that it will do nothing unless in combat mode.